contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+RUN dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony; dnf -y clean all
+RUN dnf -y install skopeo buildah; dnf -y clean all
+RUN systemctl enable podman.service
+RUN printf "\nconfdir /etc/chrony.d\n" >> /etc/chrony.conf
+
+COPY 10-inotify-instances.conf /usr/etc/sysctl.d/10-inotify-instances.conf
+COPY 50-podman-makestep.conf /usr/etc/chrony.d/50-podman-makestep.conf
+COPY 999-podman-machine.conf /usr/etc/containers/registires.conf.d/999-podman-machine.conf
+COPY containers-etc.conf /usr/etc/containers/containers.conf
+COPY delegate.conf /usr/etc/systemd/system/user@.service.d/delegate.conf
+COPY docker-host.sh /usr/etc/profile.d/docker-host.sh
+COPY podman-docker.conf /usr/etc/tmpfiles.d/podman-docker.conf
+COPY subuid /usr/etc/subgid
+COPY subuid /usr/etc/subuid
+RUN groupadd -g 501 core; useradd -u 501 -g 501 core
+RUN --mount=type=tmpfs,destination=/var ostree container commit
+
+USER core
+RUN systemctl --user enable podman.service
+COPY linger-example.service /home/core/.config/systemd/user/linger-example.service
+COPY containers-home.conf /home/core/.config/containers/containers.conf
+
+RUN mkdir -p /home/core/.config/systemd/user/default.target.wants; \
```

Comment:
What I'd recommend instead of this is to write these files into `/etc/skel` - and have the user created on firstboot via systemd-sysusers (per above).

Reply:
There is no support for /etc/subuid and /etc/subgid there though?  Or just create those files and let systemd-sysusers create it.

Reply:
Now using /etc/skel